messages/install.txt

```diff
@@ -7,4 +7,4 @@ This linter plugin for SublimeLinter provides an interface to roslint.
 Before this plugin will activate, you *must*
 follow the installation instructions here:
 
-https://github.com/lucbettaieb/SublimeLinter-roslint
+https://github.com/Sotilrac/SublimeLinter-roslint
```

Comment:
This is a fork, no?  Mind leaving it upstream?

Reply:
Yup, sounds good, I just needed it for ppl to install but I rather get it merged.